WebRoot/img/zimbra/ImgLeftArrow.svg

```diff
@@ -0,0 +1,3 @@
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="32" height="32" viewBox="0 0 32 32">
```

Comment:
if possible remove height/width, as we are not using it and it becomes easier to identify icon files in file preview